mgmtsystem_review/readme/USAGE.md

```diff
@@ -0,0 +1,9 @@
+To use this module, you need to:
+
+- go to Management Systems / Reviews
+- create a new review with the date, the persons, the policies of your
+  systems, the KPI and survey results
+- Discuss each problems and log your decision in the review lines with
+  an action or nonconformity
+- Conclude your review with the date of the next one and save it
+- Print the report
```

Comment:
tried to test on runbot, cant see anywhere action to print report? Either looking on wrong place or there is some access right issue?

Reply:
I investigated and it seems that the report from this module has been outdated since version 13.

Reply:
what do you mean by "outdated"? Code for report is included in PR, i believe fetched from previous version, i'd simply try to make it working.

Reply:
- To make the "Print Report" button visible, you need to add the following lines to `review_report_mgmtsystem_review`:
```
<field name="binding_model_id" ref="your_model_id"/>
<field name="binding_type">report</field>
```

- When clicking "Print Report," an error will appear. The issue arises from changes in the fields of the survey.user_input.line model from version 13.0 to 14.0:

  - `value_text` was changed to `value_char_box`
  - `value_free_text` was changed to `value_text_box`

 
I have updated these fields and tested the changes, but it didn’t work. I am continuing to investigate the issue.

Reply:
Hi @ajaniszewska-dev , the report is working well and ready for test
![image](https://github.com/user-attachments/assets/82725f39-7484-437c-8ecb-30df36f1f0e0)
![image](https://github.com/user-attachments/assets/8a55e6df-3932-42e2-984d-f9abed8f6432)